### ISSUE TITLE
[Feature request] Option to auto generate regularization images for training data set

### ISSUE THREAD

**Issue by StableLlama** [bmaltais/kohya_ss#1989]
Disclaimer: I'm still very new to the AI image generation topic, but reading and trying stuff myself I learned a little helpful trick:

To make a LoRA training focus on the training images and not learn something else that it's not supposed to do (optimizers like to take shortcuts) it's helpful to have regularization images. And it works best when these images have exactly the same prompt as the training images with only the training keyword replaced by the class key word. Also they should be generated exactly like the trainer is working internally (same seed and same sampling method).

### So I have this feature request:

Have an option that takes all training image prompts and creates with the same seed that is set for the training and with the DDIM sampler for each prompt an image that is stored in the regularization folder. In the used prompt the keyword is replaced with the class word.

**Example:**
When in the file `.../lora/img/10_VirtualAnna woman/portrait12.txt` the caption

> VirtualAnna portrait, smiling, white background, photo studio, photography

is stored and this tool is run then it should generate the file `.../lora/regularization/1_woman/portrait12.txt` with the content

> woman portrait, smiling, white background, photo studio, photography

and create the image with this prompt, the DDIM sampler and the same seed used for the training.

**Note:**
It is easy to do that manually but an automation would be appreciated.

**Comment by StableLlama** [comment]
@bmaltais thank you. But I couldn't find any commit or documentation about how to autogenerate the regularization images. I also didn't find any option when opening the GUI.

How is this supposed to work / to be used now?
